### ISSUE TITLE
Changing Font Size

### ISSUE THREAD

**Issue by Freudentaumel** [brief-rss/brief#409]
How can I change the font size in the right pane (this is the news pane)?

**Comment by sleek22** [comment]
Place the following in Brief's "Customize Brief's look" under options. The change is immediate if you do it correctly.

`.content {font-size: 20px !important}`

For the feed titles, use the CSS selector ".title-link"

**Comment by Freudentaumel** [comment]
There's one more question I'd like to ask you:

Is there also a CSS code for the left column (feed list) with which I can adjust the font size?

I would like that because I could adjust the font sizes of the feed list, the title and the news independently.

**Comment by sleek22** [comment]
A quick search through the Brief add-on code and I found this which worked:

`label.title {font-size: 20px !important`

I don't have a list of CSS values. In the past, I simply looked in the Brief add-on using 7-zip to extract and searching the files with the ending ".css". The above, for example, I found by trying various items in the "feedlist.css" file until I found what worked. The good thing is that you don't have to restart Firefox every time you want to try a different code because you're making those changes in the Brief options page itself.

Now normally, I just use Browser Toolbox (in Web Developer) to find the ids of elements in Firefox but it doesn't seem to work for the items in Brief. Otherwise, I could just mouse over the elements and get the ids that way. Which is why I have to do it by trial and error using the Brief code in the add-on.
